### ISSUE TITLE
Create App

### ISSUE THREAD

**Issue by pratikbutani** [pratikbutani/Men-GitHubers#16]
Create simple two page application to display all contributors list and their profile.

So everyone can follow easily and get help too.

**Comment by DearDhruv** [comment]
I would suggest creating flutter app for it. (I'm no way expert in #flutter).
It should reduce time for development.

**Comment by pratikbutani** [comment]
> I would suggest creating flutter app for it. (I'm no way expert in #flutter).
> It should reduce time for development.

Yeah It would be great. Recommend some professionals.  👍

**Comment by DearDhruv** [comment]
@kushal2011

**Comment by kushal2011** [comment]
Sounds cool , i will need an api to fetch this data

**Comment by pratikbutani** [comment]
> Sounds cool , i will need an api to fetch this data

Can we use Firebase Firestore as backend? It will be helpful as we dont need API and server too.

**Comment by kushal2011** [comment]
Yes that would be even better...
Just one question though how will the data be added into firebase....?
Can someone write a (type/java)script code to get list from the repo and add it to firebase database....?

**Comment by pratikbutani** [comment]
> Yes that would be even better...
> Just one question though how will the data be added into firebase....?
> Can someone write a (type/java)script code to get list from the repo and add it to firebase database....?

I will do, I will create web page to add easily in database. you can start with temp data. @kushal2011

**Comment by kushal2011** [comment]
Awesome 👍
add me in firebase project ,
I will create a datamodel by tonight

**Comment by ravirupareliya** [comment]
> I will do, I will create web page to add easily in database. you can start with temp data.

Instead of creating web page/manual entry, we should learn automation. We can read PR content and add it to the firestore using cloud functions. I am interested in it. Let's do some research and initiate this task instead of rush towards the application and build something from which we can not learn anything.

Opinions are always welcome, I just want everyone to learn something new instead of showing the things which we already knew :)

**Comment by pratikbutani** [comment]
@ravirupareliya Yeah You are right. I thought that once app will be created we can make it automatic.

Meanwhile I will try to find out solution to make it automated system.

**Comment by pratikbutani** [comment]
@ravirupareliya Hope [this](https://github.community/t5/How-to-use-Git-and-GitHub/Read-PR-using-bot-and-do-operation-on-that-content/m-p/34991) or [this](https://stackoverflow.com/questions/58431074/read-pr-using-bot-and-do-operation-on-that-content) will get answer, still I am trying to find way for automation.

**Comment by pratikbutani** [comment]
@ravirupareliya @mayursojitra @DearDhruv @kushal2011 
I am exploring following references and trying to make this repo can run automatically.
- [GitHub Actions](https://help.github.com/en/github/automating-your-workflow-with-github-actions/about-github-actions)
- [Action Firebase](https://natemoo.re/posts/action-firebase)
- [life-of-a-github-action](https://blog.jessfraz.com/post/the-life-of-a-github-action/)
- [Awesome-Pull-Request](https://github.com/sdras/awesome-actions#pull-requests)

You can also check that If you are free in vacation time. 📦 😄

**Comment by pratikbutani** [comment]
> @ravirupareliya @mayursojitra @DearDhruv @kushal2011
> I am exploring following references and trying to make this repo can run automatically.
> 
> * [GitHub Actions](https://help.github.com/en/github/automating-your-workflow-with-github-actions/about-github-actions)
> * [Action Firebase](https://natemoo.re/posts/action-firebase)
> * [life-of-a-github-action](https://blog.jessfraz.com/post/the-life-of-a-github-action/)
> * [Awesome-Pull-Request](https://github.com/sdras/awesome-actions#pull-requests)
> 
> You can also check that If you are free in vacation time. 📦 😄

So will you go with this?

**Comment by DearDhruv** [comment]
let's start.

**Comment by pratikbutani** [comment]
@ravirupareliya Firebase cloud function is working to store data automatically?

**Comment by mayursojitra** [comment]
Yo Aaahhu
